src/app/start-page/components/start-page-search/start-page-search.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="start-page-search">
```

Comment:
Sorry, found another weird behavior 🙈
1. Go to the map
2. Search for `al`
3. Wait for the addresses and maps to appear
4. Press `TAB` until you reach the first map. As you can see there is no preview in the background (temporary map) even if there should be one.
![image](https://github.com/user-attachments/assets/72060107-0467-42cb-8b1f-eee6a79ccf20)
5. Press `DOWN` once. You'll land on the first address result. Which is not something I would expect.
![image](https://github.com/user-attachments/assets/ee174cc8-8e1d-4d64-9e57-2a8191e14fed)